src/express/commands/start.js

```diff
@@ -134,7 +147,12 @@ async function installHostSpecificPackages(ip) {
     await runSshCommand(ip, command, maxRetries)
 }
 
-async function installDocker(ip, user) {
+async function installDocker(ip) {
```

Comment:
can you pass the user here as before, to avoid a re-split?

Reply:
Done: [a2b9787](https://github.com/maticnetwork/matic-cli/pull/82/commits/a2b978715cd75135ede2db6185ff6f395f4e8c15)